WordPress/src/main/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentViewHolder.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.sitecreation.segments
+
+import android.support.annotation.LayoutRes
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.vertical.VerticalSegmentModel
+import org.wordpress.android.util.StringUtils
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType.IMAGE
+
+sealed class NewSiteCreationSegmentViewHolder(internal val parent: ViewGroup, @LayoutRes layout: Int) :
+        RecyclerView.ViewHolder(LayoutInflater.from(parent.context).inflate(layout, parent, false)) {
+    abstract fun onBind(model: VerticalSegmentModel, isLast: Boolean)
```

Comment:
This is completely fine for this feature, especially right now when we don't have any details in the model anyway. However, I'd really love to see an adapter item that we map to instead of using the model directly if the model ever becomes complicated. Just something to keep in mind for future.

Reply:
I was thinking about creating something like `ItemUiState` with visibility states + onClickLambda + data. Is this something you have in mind?

Reply:
It could either have `VerticalSegmentModel` in an internal field and expose only required data or it could have separate fields and we'd map the data in the VM.